### DISCUSSION TITLE
Can I use 2 code locations from 2 differents branch ?

### DISCUSSION THREAD

**Discussion by dridk** [dagster-io/dagster#19705] [Q&A]
Hi,

From dagit, can I use 1 code location for a "dev" git branch and 1 code location for a  git "test" branch  ? 

I mean : Are code locations independant from each other ?

**Comment by garethbrickman** [comment]
It sounds like what you're after is [Branch Deployment, a feature we have in Dagster Cloud](https://docs.dagster.io/dagster-cloud/managing-deployments/branch-deployments#setting-up-branch-deployments): "For every push to a branch in your git repository, Dagster Cloud will create a unique deployment, allowing you to preview the changes in the branch in real-time."
